app/controllers/events_controller.rb

```diff
@@ -1,25 +1,45 @@
 # frozen_string_literal: true
 
+require 'uri'
```

Comment:
Czy to jest potrzebne? Nie widze kodu, który by korzystał z `URI`. Rubocop exclude też chyba nie jest potrzebny

Reply:
OK, już widzę, że chodziło o `host/events_controller.rb`